src/lib/awsServicesConnector.ts

```diff
@@ -0,0 +1,20 @@
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { DynamoDBDocumentClient } from "@aws-sdk/lib-dynamodb";
+import { AWS_REGION, LOCALSTACK_ENDPOINT } from "@src/config";
+
+const globalConfig = {
+  region: AWS_REGION,
+};
+
+const localstackConfig = {
+  ...(LOCALSTACK_ENDPOINT && {
+    endpoint: LOCALSTACK_ENDPOINT,
+  }),
+};
+
+export const dynamodbClient = DynamoDBDocumentClient.from(
```

Comment:
Does it make sense to initialize the AWS clients once and then reuse them instead of initializing on every invocation?  I noticed we could do the same potential optimization in the GC Forms web app code.

Reply:
100%. I will modify it in both projects ;)

Reply:
@bryan-robitaille I said I agree that we should be doing just one instance but it is in fact what is already happening here.
I found the pull request I created in the web app a while ago to make this `awsServicesConnector` refactoring thing. Here it is https://github.com/cds-snc/platform-forms-client/pull/3571. In the title we can see that the goal was to have just one instance of the AWS SDK.

I can confirm there is only one instance created with this code.

Reply:
Right, exporting an object instead of a function ensures it is cached by Node.  I'm so used to seeing arrow functions my brain inserted `export const dynamodbclient = () => DynamodbClient.from ...`